openapi/src/url_finder.rs

```diff
@@ -24,8 +25,9 @@ impl UrlFinder {
     }
 
     pub fn url_for_object(&self, object: &str) -> Option<String> {
+        let args = Command::parse();
         let unprefixed_link = self.doc_links.get(object)?;
-        Some(format!("https://stripe.com/docs/api{}", unprefixed_link))
+        Some(format!("{}{}", args.api_docs_url, unprefixed_link))
```

Comment:
There was a bunch of churn at some point around how Stripe structured their docs and a bunch of the handling for this stuff changed. I think this argument is a relic of that and the option can just be safely removed